docs/apps/headphones.md

```diff
@@ -1 +1,7 @@
-Placeholder page
+# Headphones
+
+[Headphones](https://github.com/rembo10/headphones) is an automated music downloader for NZB and Torrent, written in Python. It supports SABnzbd, NZBget, Transmission, µTorrent and Blackhole.
+
+The GIT Repository for Headphones is located at [https://github.com/linuxserver/docker-headphones](https://github.com/linuxserver/docker-headphones).
+
+
```

Comment:
Please remove the extra newline at the end of the file (one newline is good, two is not needed).